package/boot/tfa-layerscape/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tfa-layerscape
-PKG_VERSION:=lf-6.1.1-1.0.0
-PKG_RELEASE:=1
+PKG_VERSION:=lf_6.6.3_1.0.0
+PKG_RELEASE:=r1
```

Comment:
The r is now automatically added via the build system, no need to define it

Reply:
I fixed all bumps.